1q

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2020 Dan Halbert for Adafruit Industries
```

Comment:
it seems like this file got added accidently or else renamed to `1q` accidentally. It contains code from `humidity_service.py`.